examples/0_basic/0_create_did.rs

```diff
@@ -1,54 +1,48 @@
-// Copyright 2020-2022 IOTA Stiftung
-// SPDX-License-Identifier: Apache-2.0
-
-use anyhow::Context;
-use examples::get_address_with_funds;
-use examples::random_stronghold_path;
-use examples::NETWORK_ENDPOINT;
-use identity_core::convert::ToJson;
-use identity_core::crypto::KeyPair;
-use identity_core::crypto::KeyType;
-use identity_did::verification::MethodScope;
-use identity_stardust::NetworkName;
-use identity_stardust::StardustClientExt;
-use identity_stardust::StardustDocument;
-use identity_stardust::StardustIdentityClientExt;
-use identity_stardust::StardustVerificationMethod;
+use identity_iota::core::ToJson;
+use identity_iota::crypto::KeyPair;
+use identity_iota::crypto::KeyType;
+use identity_iota::did::MethodScope;
+use identity_iota::iota_core::IotaClientExt;
+use identity_iota::iota_core::IotaDocument;
+use identity_iota::iota_core::IotaIdentityClientExt;
+use identity_iota::iota_core::IotaVerificationMethod;
+use identity_iota::iota_core::NetworkName;
 use iota_client::block::address::Address;
 use iota_client::block::output::AliasOutput;
 use iota_client::secret::stronghold::StrongholdSecretManager;
 use iota_client::secret::SecretManager;
 use iota_client::Client;
 
+// The endpoint of the IOTA node to use.
+static NETWORK_ENDPOINT: &str = "https://127.0.0.1:14265";
+
 /// Demonstrates how to create a DID Document and publish it in a new Alias Output.
 #[tokio::main]
-async fn main() -> anyhow::Result<()> {
+async fn main() -> Result<(), Box<dyn std::error::Error>> {
```

Comment:
Is there a reason we don't use `anyhow::Result` here but we do in other examples?

Reply:
Updated example to use `anyhow`.